qiskit_experiments/test/Fine Calibration Tutorial.rst

```diff
@@ -0,0 +1,154 @@
+
+Let's start with importing some necessary modules and build our pulse simulator with
+"SingleTransmonTestBackend" which is inherited from "IQPulseBackend"
+
+.. jupyter-execute:: 
+
+    import numpy as np
+    from qiskit.pulse import InstructionScheduleMap
+    import qiskit.pulse as pulse
+    from qiskit_experiments.library import FineXAmplitude, FineSXAmplitude
+    from qiskit_experiments.test.iq_pulse_backend import SingleTransmonTestBackend
+
+.. jupyter-execute::
+
+    backend = SingleTransmonTestBackend()
+    qubit = 0
+-----------------------------------------------------
+Fine X gate Amplitude Calibration
+-----------------------------------------------------
+We will run the fine calibration experiments with our own pulse schedules. 
+To do this we create an instruction to schedule map which we populate with 
+the schedules we wish to work with. This instruction schedule map is then 
+given to the transpile options of the calibration experiments so that 
+the Qiskit transpiler can attach the pulse schedules to the gates in the experiments. 
+We will base all our pulses on the default X pulse of "SingleTransmonTestBackend".
+
+.. jupyter-execute::
+    x_pulse = backend.defaults().instruction_schedule_map.get('x', (qubit,)).instructions[0][1].pulse
+    d0, inst_map = pulse.DriveChannel(qubit), InstructionScheduleMap()
+
+    for name, pulse_ in [("x", x_pulse)]:
+        with pulse.build(name=name) as sched:
+            pulse.play(pulse_, d0)
+
+        inst_map.add(name, (qubit,), sched)
+
+We now take the ideal x pulse amplitude reported by the backend and 
+add/subtract a 2% over/underrotation to it by scaling the ideal amplitude and see 
+if the experiment can detect this over/underrotation. We replace the default X pulse 
+in the instruction schedule map with this over/underrotated pulse.
+
+.. jupyter-execute::
+    ideal_amp = x_pulse.amp
+    over_amp = ideal_amp*1.02
+    under_amp = ideal_amp*0.98
+    print(f"The reported amplitude of the X pulse is {ideal_amp:.4f} which we set as ideal_amp. we use {over_amp:.4f} amplitude for overroation pulse and {under_amp:.4f} for underrotation pulse")
+
+-----------------------------------------------------------------------------------
+Detecting an over & under rotated pulse
+-----------------------------------------------------------------------------------
+
+.. jupyter-execute::
+    # build the over rotated pulse and add it to the instruction schedule map
+    with pulse.build(backend=backend, name="x") as x_over:
+        pulse.play(pulse.Drag(x_pulse.duration, over_amp, x_pulse.sigma, x_pulse.beta), d0)
+    inst_map.add("x", (qubit,), x_over)
+
+    # do the experiment
+    overamp_cal = FineXAmplitude(qubit, backend=backend)
+    overamp_cal.set_traspile_options(inst_map=inst_map)
+    exp_data_over = overamp_cal.run(backend)
+    exp_data_over.figure(0)
+
+.. jupyter-execute::
```

Comment:
It is good that you do both over and under rotation of the gates. However, you need to comment on the figures that are being shown. Something like saying that the ping-pong pattern that we see is flip depending on whether we have an under or an over rotation. Basically we need a small piece of text to explain to the reader what she/he is seeing.